biotope/2018-06-05_biotopeVision.md

```diff
@@ -0,0 +1,20 @@
+# Biotope is a framework which enables you to develop scalabale, platform-agnostic frontend component ecosystems
+
+## It helps developers to:
+- easily develop ui-components following a unified standard
+- keep focus on developing components, the architecture and dev environment is provided by Biotope.
+- build platform-agnostic ui components which can be used in single page applications as well as in traditional server side rendered cms projects
+- make it easy to work with a team of many developers, all providing components to the library
+- easily ensure high quality standards for the whole library even when developed by multiple suppliers
+- build real projects and design system styleguides with a single code source
+- create your own ui library to choose from in your projects
+
+
+## It helps companies to:
+- Create a UI component library which can be maintained in a central place
+- BUT all ui-components can be used independently in different projects
+- Create UI consistency over all projects
+- Make it possbile to let multiple partners contribute to the ui library without compatiblity & quality issues
```

Comment:
"Have/Let multiple partners contribute to the UI library without compatibility & quality issues"